src/styles/style.scss

```
@@ -484,7 +484,7 @@ a {
 }
 
 .news_post:first-child div{
-  background-image: url(../images/widget_post.png);
+  background-image: url(../images/Medium\ Visual.png);
   background-size: cover;
```

The visual seems a bit small. We can make it larger by increasing the `background-size` and then center it:
```suggestion
  background-size: 150%;
  background-position: center;
```